lib/common_test/doc/src/ct_suite.xml

```diff
@@ -604,6 +604,9 @@
 	<p>To print some information in field <c>Comment</c> on the HTML
 	  result page, return <c>{comment, Comment}</c>.</p>
 
+        <p>If the function returns <c>{fail, Reason}</c>, the test case
+          is considered as failed.</p>
+
 	<p>If the function returns anything else, the test case is
```

Comment:
While reviewing this part, maybe we could improve clarity here as well ... I hope :-)
It think this is not as clear about `{comment, Comment}` as it should/could be. What do you think?

1. `{comment, Comment}` paragraph is not explicitly suggesting "The test case is considered successful, but with a comment."
2. "If the function returns _anything else_, the test case is considered successful.". When people learn and can't absorb all information or just skim docs, I'm afraid they might draw a conclusion that a testcase returning `{comment, Comment}` will not be considered successful.

Maybe I'm picky here, but I had students asking those questions :-) It would be great if I could redirect them to clear docs instead of explaining it myself ;-).